### ISSUE TITLE
9.2.4.7 Aktuelle Position des Fokus deutlich - Kontrastansicht

### ISSUE THREAD

**Issue by sweckenmann** [BIK-BITV/BIK-Web-Test#14]
Der Kontrast einer grafischen Fokushervorhebung (z.B. ein farbiger Rahmen um einen Link) und auch der Farbabstand bei Fokushervorhebung nur über Farbwechsel (häufiger bei Button) muss 3,0:1 erfüllen. (Wir hatten uns ja geeinigt, Mängel bzgl. Fokushervorhebung bei 9.2.4.7 zu dokumentieren, auch wenn die Anforderung streng genommen Gegenstand von 9.1.4.11 ist).

Es stellte sich die Frage, ob im Falle einer Kontrastansicht die Kontraste dieser Fokushervorhebung nicht eigentlich in beiden Ansichten geprüft werden müssten (Standard- und Kontrastansicht), denn sehende Tastaturnutzer:innen werden vermutlich nicht unbedingt die Kontrastansicht aufrufen und benötigen auch in der Standardansicht einen deutlichen Fokus?! 

Das kann jedoch, je nach Umsetzung, ein deutlicher Mehraufwand bedeuten.

**Comment by johannesFischer84** [comment]
> [...] und auch der Farbabstand bei Fokushervorhebung nur über Farbwechsel (häufiger bei Button) muss 3,0:1 erfüllen.

Der Kontrast von 3:1 des Unterschieds unfokussiert - fokussiert scheint mir hier nur eine Möglichkeit der Erfüllung von 9.2.4.7 zu sein. Wenn gleichzeitig der Standardfokus des Browsers normal sichtbar ist, wäre 2.4.7 erfüllt, auch wenn der Unterschied einer farblichen Veränderung im Fokusfall nicht groß wäre.

Aber zur eigentlichen Frage: Ich habe es so verstanden, dass normal gut sehende Tastaturnutzer nicht die kontrastreiche Ansicht aufrufen, weil sie in der Regel ja alles gut sehen können. Das heißt, in der normalen Ansicht darf der normale Fokus nach 9.2.4.7 nicht blockiert sein und ein selbst gestalteter Fokus muss nach 9.1.4.11 (oder meinetwegen 9.2.4.7) mindestens 3:1 nach außen erreichen. Die kontrastreiche Ansicht darf dem natürlich in nichts nachstehen.

Also ja, ich würde dir zustimmen, Sonja. Wobei es auch sein könnte, dass die Fokushervorhebung sowieso erst in der kontrastreichen Ansicht zu sehen ist, in der Standard-Ansicht dagegen nur der Standard-Fokus des Browsers. Aber den Prüfaufwand reduziert das natürlich nicht.

**Comment by sweckenmann** [comment]
> Der Kontrast von 3:1 des Unterschieds unfokussiert - fokussiert scheint mir hier nur eine Möglichkeit der Erfüllung von 9.2.4.7 zu sein. Wenn gleichzeitig der Standardfokus des Browsers normal sichtbar ist, wäre 2.4.7 erfüllt, auch wenn der Unterschied einer farblichen Veränderung im Fokusfall nicht groß wäre.

Richtig, sobald eine zusätzliche Hervorhebung hinzukommt, Systemkranz oder zusätzliche Unterstreichung ist der Abstand nicht relevant. Nur wenn der Farbwechsel das einzige Mittel der Fokushervorhebung ist, kommt die Kontrastanforderung zum Tragen.

**Comment by detlevhfischer** [comment]
Also Prüfschrittanleitung ergänzen im Snne von "Falls eine Kontrastansicht geboten wird, muss auch diese eine deutliche Tastatur-Fokushervorhebung aufweisen..."?
